bindings/python/bt2/bt2/packet.py

```diff
@@ -33,7 +34,7 @@ class _Packet(object._Object):
     def stream(self):
         stream_ptr = native_bt.packet_get_stream(self._ptr)
         assert(stream_ptr)
-        return bt2.stream._Stream._create_from_ptr(stream_ptr)
+        return bt2.domain._Domain.create_stream_from_ptr(stream_ptr)
```

Comment:
Not sure about this. Why not using the IR domain's `create_stream_from_ptr()`?

Reply:
That's what it's using. I added an explicit import of the IR domain module.